### DISCUSSION TITLE
URL Alias einbauen

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/datalinq-community#14] [Ideas]
Hallo!

Ich hätte einen feature request:

Wir basteln gerade an einem Report zum Thema Badegewässer Beprobungen. Das wäre der derzeitige Aufruf:
TEST:https://... .at/**webgis5_api/datalinq/report/p-baden@badestellen@view**

Ich würde mir wünschen, dass die produktive Alias URL dann so aussieht:
https:/atlas.vorarlberg.at/**badestellen**

Ist das machbar? Bin da leider auch zu wenig firm in dem Thema...

**Comment by gis-eni** [comment]
Ist meiner Meinung keine WebGIS oder DataLinq Thema. Da musst du dich eher an die IT wenden. Reverse Proxy? NetScaler?

**Reply by ue-black** [reply]
Unsere interne IT hat zuerst auf die Anwendung verwiesen und nach nachbohren meinerseits haben sie jetzt doch eine Lösung für mich gefunden...
